player/low-latency-streaming/js/script.js

```diff
@@ -170,4 +176,234 @@ function getQueryParams() {
+}
+
+function updateCharts(player) {
+  addNewData(player.getVideoBufferLength(), player.getAudioBufferLength(), player.getDownloadedVideoData().bitrate);
+}
+
+function addChartData(chart, seriesIndex, xAxis, yAxis) {
+  chart.series[seriesIndex].addPoint([xAxis, yAxis], true, false);
+}
+
+function setupChart() {
+  initialTimestamp = Date.now();
+  bufferChart = Highcharts.chart(document.getElementById("buffer-chart"), {
+
+    chart: {
+      type: 'spline',
+      zoomType: 'x'
+    },
+    credits: {
+      enabled: false
+    },
+    title: {
+      text: 'Buffer Levels'
+    },
+    xAxis: {
+      title: {
+        text: 'time',
+        align: 'low'
+      },
+      min: 0
+    },
+    yAxis: {
+      title: {
+        text: 'sec',
+        align: 'high'
+      },
+      min: 0
+    },
+    legend: {
+      align: 'center',
+      verticalAlign: 'bottom'
+    },
+    series: [{
+      name: 'Video',
+      data: [[0, 0]],
+      marker: {
+        enabled: true,
+        fillColor: '#ffffff',
+        lineWidth: 2,
+        lineColor: null,
+        symbol: 'circle'
+      },
+      color: '#1FAAE2'
+    }, {
+      name: 'Audio',
+      data: [[0, 0]],
+      marker: {
+        enabled: true,
+        fillColor: '#ffffff',
+        lineWidth: 2,
+        lineColor: null,
+        symbol: 'circle'
+      },
+      color: '#F49D1D'
+    }],
+
+    responsive: {
+      rules: [{
+        condition: {
+          maxWidth: 500
+        },
+        chartOptions: {
+          legend: {
+            layout: 'horizontal',
+            align: 'center',
+            verticalAlign: 'bottom'
+          }
+        }
+      }]
+    }
+  });
+
+  bitrateChart = Highcharts.chart(document.getElementById("bitrate-chart"), {
+    chart: {
+      type: 'spline',
+      zoomType: 'x'
+    },
+    credits: {
+      enabled: false
+    },
+    title: {
+      text: 'Bitrate'
+    },
+    xAxis: {
+      title: {
+        text: 'time',
+        align: 'low'
+      },
+      min: 0
+    },
+    yAxis: {
+      title: {
+        text: 'Mbps',
+        align: 'high'
+      },
+      min: 0
+    },
+    legend: {
+      align: 'center',
+      verticalAlign: 'bottom'
+    },
+    series: [{
+      name: 'Video',
+      data: [[0, 0]],
+      marker: {
+        enabled: true,
+        fillColor: '#ffffff',
+        lineWidth: 2,
+        lineColor: null,
+        symbol: 'circle'
+      },
+      color: '#1FAAE2'
+    }],
+    responsive: {
+      rules: [{
+        condition: {
+          maxWidth: 500
+        },
+        chartOptions: {
+          legend: {
+            layout: 'horizontal',
+            align: 'center',
+            verticalAlign: 'bottom'
+          }
+        }
+      }]
+    }
+  });
+}
+
+function setPlayerEvents(player) {
+    player.on(bitmovin.player.PlayerEvent.AudioPlaybackQualityChanged, function (data) {
+      log("On Audio Playback Quality Changed: " + JSON.stringify(data));
```

Comment:
this actually causes an exception because there is no `log` function. So the `updateCharts(player);` will not be called here.
So either you remove the logs or you add the log function, something like:
```js
function log(text) {
  console.debug(text);
}
```